### PR TITLE
MAUI-845777 : [MAUI - SfPopup] Prepare UG for popup static and Async methods that returns result

### PR DESCRIPTION
**Pull request by JayashreeRavishankar**
Prepared UG content for or popup static and Async methods that returns result

### CODE REVIEW

File: MAUI/Popup/popup-positioning.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ Following are the list of options available to show SfPopup at various positions
 <tr>
 <td>{{'[Show(bool)](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Popup.SfPopup.html#Syncfusion_Maui_Popup_SfPopup_Show_System_Boolean_)'| markdownify }}</td>
 <td>Similar as <code>SfPopup.IsFullScreen</code> property.</td></tr>
+<tr>
+<td><code>SfPopup.Show(string title, string message)</code></td>
+<td>Static method to show popup with title and message.</td></tr>
```
Comment (Chrispineagunja) [inline MAUI/Popup/popup-positioning.md:43]: <td>Static method to show a popup with the title and message.</td></tr>

File: MAUI/Popup/popup-positioning.md [hunk 2]
```diff
@@ -38,6 +38,12 @@ Following are the list of options available to show SfPopup at various positions
 <tr>
 <td>{{'[Show(bool)](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Popup.SfPopup.html#Syncfusion_Maui_Popup_SfPopup_Show_System_Boolean_)'| markdownify }}</td>
 <td>Similar as <code>SfPopup.IsFullScreen</code> property.</td></tr>
+<tr>
+<td><code>SfPopup.Show(string title, string message)</code></td>
+<td>Static method to show popup with title and message.</td></tr>
+<tr>
+<td><code>SfPopup.Show(string title, string message, string acceptText)</code></td>
+<td>Static method to show popup with title, message and acceptText.</td></tr>
```
Comment (Chrispineagunja) [inline MAUI/Popup/popup-positioning.md:46]: <td>Static method to show a popup with the title, message, and acceptText.</td></tr>

File: MAUI/Popup/popup-positioning.md [hunk 3]
```diff
@@ -268,6 +274,80 @@ public class ViewModel : INotifyPropertyChanged
...
+  {
+    bool answer = await SfPopup.Show("Question?", "Would you like to play a game", "Yes", "No");
+  }
+}
+{% endhighlight %}
+{% endtabs %}
+
 ### Load the SfPopup in CellTappedEvent of the SfDataGrid
 
 The `SfPopup` allows opening it in the [CellTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CellTapped) event of the `SfDataGrid`.
```
Comment (Chrispineagunja) [inline MAUI/Popup/popup-positioning.md:353]: The `SfPopup` allows opening it in the [CellTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CellTapped) event of the `SfDataGrid.`

**Review by Chrispineagunja — COMMENTED** [review]
Content review completed.

**Review by PiruthivirajM — APPROVED** [review]

**Pull request merged**